### ISSUE TITLE
Expand on frame size computation

### ISSUE THREAD

**Issue by wilbowma** [cpsc411/cpsc411-book#49]
>  M6E11 Confused about the wording
> 
> In the textbook it says
> 
> > The size n is one plus the maximum of the index of all frame variables in the call-undead set for the enclosing block.
> 
> Isn't this the same as the length of call-undead set?

This confused students. We need to expand and give examples of when frame vars are live across a return point. This can happen with tail calls that have many arguments, or when a call-undead variable is assigned to a out-of-sequence frame variable.

**Comment by wilbowma** [comment]
Revisit this after #34, which should enable fvars to appear syntactically in the source. See also #59

**Comment by wilbowma** [comment]
See also #42

**Comment by wilbowma** [comment]
Duplicate of #42
